app/extensions/brave/content/scripts/passwordManager.js

```diff
@@ -74,11 +74,19 @@ if (chrome.contentSettings.passwordManager == 'allow') {
             }, usernameElem.value || '')
         }
       })
+      usernameElem.addEventListener('blur', (e) => {
```

Comment:
STR:
go to github.com in a new private tab
start entering previously-saved username
click on the context menu that pops up

for me, the entry is not filled at that point. may be a race condition.